src/DocFxOpenApi/DocFxOpenApi.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Licensed to DocFX Companion Tools and contributors under one or more agreements.
+// DocFX Companion Tools and contributors licenses this file to you under the MIT license.
+
+namespace DocFxOpenApi
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using CommandLine;
+    using global::DocFxOpenApi.Domain;
+    using global::DocFxOpenApi.Helpers;
+    using Microsoft.OpenApi;
+    using Microsoft.OpenApi.Extensions;
+    using Microsoft.OpenApi.Models;
+    using Microsoft.OpenApi.Readers;
+
+    /// <summary>
+    /// Open API file converter to V2 JSON files.
+    /// </summary>
+    internal class DocFxOpenApi
+    {
+        private const OpenApiSpecVersion _outputVersion = OpenApiSpecVersion.OpenApi2_0;
```

Comment:
keep the constants as they were but please change the private elements with the underscore pattern and remove the `this.` which are not necessary